lib/widgets/classify_species.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:http/http.dart';
+import 'package:flutter/services.dart' show rootBundle;
+
+
+class SpeciesClassifier {
+  var queryParameters = {
+    'topK': '1',
+    'predictMode': 'classifyAndDetect',
+  };
+  Map<String, String> headers = {
+    'Content-Type': 'application/octet-stream',
+    'Ocp-Apim-Subscription-Key': '485afbef61f9444bbcb4b454ef35d966'
```

Comment:
We shouldn't store keys like this in version control (cause that means anyone can make requests using it and can rack up billing charges), so we need to refactor this out or encode it somehow in the future

Reply:
That's what I was thinking but I have no idea where to actually store it, I will look into it tho

Reply:
We can figure this out after tomorrow's demo